test/e2e/node_exporter_test.go

```diff
@@ -126,6 +126,14 @@ nodeExporter:
 				"node_network_transmit_packets_total",
 			},
 		},
+		{
+			nameCollector: "netclass",
```

Comment:
we need another test asserting that the netclass metrics exist in the default config.

Reply:
metrics check is added now.